CMakeLists.txt

```diff
@@ -78,8 +78,17 @@ ENDIF()
 ##---------------------------------------------------------------------------##
 
 ADD_TEST(
-    NAME check_no_trailing_whitespace
-    COMMAND sh -c "grep --exclude=*.git* -rl '[[:blank:]]$' ${CMAKE_SOURCE_DIR} test $? -eq 1"
+    NAME check_no_trailing_whitespace_nor_tab
+    COMMAND sh -c "grep \
+         --exclude-dir={.git,doc} \
```

Comment:
I think that's where the error is.  I tested with `.` in place of `${${PACKAGE_NAME}_SOURCE_DIR}`.